### PR TITLE
ClamAV - add a mechanism to inform about found files

### PR DESCRIPTION
**Pull request by szaimen**
Todo: 
- [x] fix all todos
- [x] escape all $ inside the cat file

### CODE REVIEW

**Comment by szaimen** [comment]
This was successfully tested with a test-virus-file hence marking as ready for review

File: apps/clamav.sh [hunk 1]
```diff
@@ -36,7 +36,9 @@ else
     rm -f /etc/systemd/system/clamav-daemon.service
     rm -f "$SCRIPTS"/clamav-fullscan.sh
     rm -f "$VMLOGS"/clamav-fullscan.log
+    rm -f "$SCRIPTS/nextcloud-av-notification.sh"
     crontab -u root -l | grep -v 'clamav-fullscan.sh'  | crontab -u root -
```
Comment (enoch85) [inline apps/clamav.sh:40]: It should still delete any old entry if this script was run previous to this change imo.

**Comment by enoch85** [comment]
I will merge when you add it to the not-supported again. :+1:

**Comment by szaimen** [comment]
> I will merge when you add it to the not-supported again. 👍

done 👍

**Pull request merged**